packages/select/test/dom/select.test.js

```diff
@@ -47,8 +47,9 @@ describe('vaadin-select', () => {
     });
 
     it('error', async () => {
+      select.required = true;
```

Comment:
There is already an individual test case for `required`. Is this change still needed then?